operator/validator/storage.go

```diff
@@ -147,6 +148,29 @@ func (s *Collection) GetOperatorValidatorShares(operatorPubKey string, enabled b
 	return res, err
 }
 
+// GetOperatorIDValidatorShares returns all not liquidated validator shares belongs to operator ID.
+func (s *Collection) GetOperatorIDValidatorShares(operatorID uint32, enabled bool) ([]*beaconprotocol.Share, error) {
```

Comment:
Not sure that we have cases where we actually need the entire share object as result (currently we just need the public key).

Not urgent as this won't happen too often, so let's add a TODO here to check regards returning a slice of public keys instead of shares